app/js/controllers/relatedCtrl.js

```diff
@@ -21,6 +22,7 @@ KMCMenu.controller('relatedCtrl', ['$scope',
 		$scope.entryListChange = function(){
 			// unselect playlist ID
 			$scope.playerData.config.plugins["related"].playlistId = null;
+			$scope.playerData.config.plugins["related"].useContext = false;
```

Comment:
move it outside the "unselect playlist ID" handler

Reply:
what do you mean, at every option selected event the rest of the options are reset , that is how it has worked before